.github/workflows/phpunit-lang.yml

```
@@ -0,0 +1,64 @@
+name: Language Unit Tests
+
+on:
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - 'src/Language/**.php'
+      - '!src/Language/en/**.php'
+      - 'phpunit*'
+      - '.github/workflows/phpunit-lang.yml'
+  push:
+    branches:
+      - develop
+    paths:
+      - 'src/Language/**.php'
+      - '!src/Language/en/**.php'
+      - 'phpunit*'
+      - '.github/workflows/phpunit-lang.yml'
+
+jobs:
+  main:
+    name: Language Unit Tests
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
```

I've started using `8.1` for our "generical" workflows:
```suggestion
          php-version: '8.1'
```